src/containers/CallIn/ThankYou/ThankYou.js

```diff
@@ -63,11 +64,41 @@ export class ThankYou extends Component {
         const trackingToken = getUrlParameter(params, 't') || undefined;
         const callerId = getUrlParameter(params, 'c') || undefined;
         this.removeTrackingGetArgs();
+        const missingParams = [
+            ['district', calledNumber],
+            ['state', calledState],
+            ['d', homeDistrictNumber],
+            ['t', trackingToken],
+            ['c', callerId]
+        ]
+        .filter(p=> !p[1])
+        .map(p=>p[0])
+        .join()
+        if (missingParams !== '') {
```

Comment:
send a breadcrumb with the status of the params one way or another. Use warning instead of info if something is missing.